packages/base/src/3dview/mainview.tsx

```diff
@@ -1323,12 +1324,17 @@ export class MainView extends React.Component<IProps, IStates> {
       const width = this.divRef.current?.clientWidth || 0;
       const height = this.divRef.current?.clientHeight || 0;
 
+      const distance = position.distanceTo(target);
+      const zoomFactor = 1000 / distance;
```

Comment:
Do you know why we need this magic `1000` number?

Reply:
This is kind of an arbitrary number after some hit and trials, I noticed that more you zoom the object in `PerspectiveCamera` more further it goes on `OrthographicCamera`

Reply:
So it meant that the distance was inversely proportional to the zooming in that i needed to do

Reply:
Sounds good! I tested on my side and this seems to work well. We have other magic numbers in JupyterCAD